ast2vec/__main__.py

```diff
@@ -65,11 +77,15 @@ def get_parser() -> argparse.ArgumentParser:
 
     bblfsh_args = argparse.ArgumentParser(add_help=False)
     bblfsh_args.add_argument(
-        "--bblfsh", default=None, dest="bblfsh_endpoint",
-        help="Babelfish server's endpoint, e.g. 0.0.0.0:9432.")
+        "--bblfsh", dest="bblfsh_endpoint",
+        help="Babelfish server's endpoint, e.g. 0.0.0.0:9432. "
+             "You can specify it directly or with BBLFSH_ENDPOINT environment variable. Otherwise "
+             "default will be used (default: {})".format(DEFAULT_BBLFSH_ENDPOINT))
```

Comment:
Please do not use `format()` - this project uses % syntax because it is the same as in other languages - Go and C/C++.

Reply:
ok!